Lesson _1_3

AlonaSamoilenko opened the pull request:
1. Moved List Item into New"TodoListItem" Component
2. Submited "AddTodoForm"
3. Stored "NewTodo" in React state

AipuAmeh commented:
I don't see where your console log prints the todoTitle in your AddTodoForm.jsx file. I also don't see a new todo printed on the page when I fill out the form. Please take another look to see what could be going wrong. If you have any questions, don't hesitate to book a 1:1 session, or attend a group session!

AlonaSamoilenko commented:
@AipuAmeh You can see console log prints the todoTitle in  AddTodoForm.jsx:7. 
I have also fixed the error which prevented a new todo printed on the page:moved submit button inside form.
